stdlib/experimental/dynamic/dynamic.flux

```diff
@@ -0,0 +1,29 @@
+// Package dynamic provides tools for working with values of unknown types.
+//
+// ## Metadata
+// introduced: NEXT
+//
+package dynamic
+
+
+// Attempts to convert a dynamic value into an array of dynamic elements.
+// If the value is not an array, `asArray` will result in a fatal error.
+//
+// ## Parameters
+// - v: Dynamic value to operate on. Defaults is the piped-forward value (`<-`).
+//
+// ## Examples
+// ### Mapping over a heterogeneous json array
+// ```
+// TODO
+// ```
+//
+// ## Metadata
+// introduced: NEXT
+builtin asArray : (<-v: dynamic) => [dynamic]
+
+// Convert a value into a dynamic.
+//
+// ## Metadata
+// introduced: NEXT
+builtin dynamic : (v: A) => dynamic
```

Comment:
Looking at some other packages in the stdlib, if there is a package-scoped `introduced` doc attribute, the package members seem inherit it if they don't have an `introduced` attribute of their own.